dist/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Also could do with a `<title>` in the header here.

Though these seem like little things, if we get them sorted from the beginning, it makes everything nice and clean and full of awesome from the start.